workspaces/local-cli/src/shared/local-cli-task-runner.ts

```diff
@@ -97,6 +102,7 @@ export async function LocalTaskSessionWrapper(
     shouldCollectCoverage: flags['collect-coverage'] !== false,
     shouldCollectDiffs: flags['collect-diffs'] !== false,
     shouldExitOnDiff: flags['exit-on-diff'] !== false,
+    shouldTransparentProxy: flags['transparent-proxy'] !== false,
```

Comment:
@LouManglass I'd confirm this logic works as intended and doesn't break people doing it the old way. 

Based on what I'm reading, it seems like anyone using the legacy ENV var will find it no longer works unless they use the flag. 

Undocumented feature, sure, so ok to change, but better if we don't have to break anyone.

Reply:
Yep, I broke that on line 250. Now it sets the variable properly when the flag is present, otherwise defaults to whatever the environment variable is (so if someone set it to `yes` in the shell, that should now carry forward). Loom to show that the flag and the environment variable work:

https://www.loom.com/share/4eaea81b9fad44beb06cf81a84244518